content/en/docs/tasks/administer-cluster/upgrade-downgrade/kubeadm-upgrade-1-11.md

```diff
@@ -0,0 +1,303 @@
+---
+reviewers:
+- pipejakob
+- luxas
+- roberthbailey
+- jbeda
+title: Upgrading kubeadm clusters from v1.10 to v1.11
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This guide is for upgrading `kubeadm` clusters from version 1.10.x to 1.11.x, as well as 1.10.x to 1.10.y and 1.11.x to 1.11.y where `y > x`.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+Before proceeding:
+
+- You need to have a functional `kubeadm` Kubernetes cluster running version 1.10.0 or higher in order to use the process described here. Swap also needs to be disabled.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md) carefully.
+- `kubeadm upgrade` now allows you to upgrade etcd. `kubeadm upgrade` will also upgrade of etcd to 3.1.10 as part of upgrading from v1.8 to v1.9 by default. This is due to the fact that etcd 3.1.10 is the officially validated etcd version for Kubernetes v1.9. The upgrade is handled automatically by kubeadm for you.
+- Note that `kubeadm upgrade` will not touch any of your workloads, only Kubernetes-internal components. As a best-practice you should back up what's important to you. For example, any app-level state, such as a database an app might depend on (like MySQL or MongoDB) must be backed up beforehand.
+
+{{< caution >}}
+**Caution:** All the containers will get restarted after the upgrade, due to container spec hash value being changed.
+{{< /caution >}}
+
+Also, note that only one minor version upgrade is supported. For example, you can only upgrade from 1.10 to 1.11, not from 1.9 to 1.11.
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Upgrading your control plane
+
+Execute these commands on your master node:
+
+1. Install the most recent version of `kubeadm` using `curl` like so:
+
+```shell
+sudo apt-get upgdate && sudo apt-get upgrade kubeadm
+```
+
+`kubeadm` is only needed on individual (non-master) nodes for joining the cluster.
+It is not necessary to update kubeadm on nodes 
+
+Verify that this download of kubeadm works and has the expected version:
+
+```shell
+kubeadm version
+```
+
+2. On the master node, run the following:
+
+```shell
+kubeadm upgrade plan
+```
+
+You should see output similar to this:
+
+<!-- TODO: copy-paste actual output once new version is stable -->
```

Comment:
Now it should kinda be from master

Reply:
What do you mean by this?

Reply:
Everything is still suffixed with `upgrading to version -beta.2.78+<hash>`, so I still want to wait.